developer-docs-site/docs/nodes/ait/additional-doc.md

```diff
@@ -4,70 +4,72 @@ slug: "additional-doc"
 sidebar_position: 15
 ---
 
-## Shutdown Nodes for Incentivized Testnet
+## Shutdown nodes for Incentivized Testnet
 
-Follow this instruction when you need to take down the validator node and cleanup the resources used by the node.
+Follow these instructions when you need to take down the validator node and cleanup the resources used by the node.
 
+Before you shutdown the node, you should make sure to leave validator set first (will take effect in next epoch).
 
-Before you shutdown the node, you should make sure to leave validator set first (will take effect in next epoch)
-
-    ```
-    aptos node leave-validator-set --profile ait3-operator
-    ```
+```bash
+aptos node leave-validator-set --profile ait3-operator
+```
 
 ### Using source code
 
 - Stop your node.
-- Remove the data directory: `rm -rf <your-data-directory>`
-- Remove the genesis blob file and waypoint
+- Remove the data directory: `rm -rf <your-data-directory>`.
```

Comment:
@gregnazario commented on my old PR that he recommends we don't have  the "force" part of this code as it often leads to issues. Idk if you wanna edit that out.

Reply:
So I am assuming it means the command should be without the `-f` flag. I will fix it now.